### PR TITLE
Fix #176642 gptchatly.com 

### LINKED ISSUE #176642: gptchatly.com
### Issue URL (Ads)
 [https://gptchatly.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fgptchatly.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.3.46
System version: | Mac OS 14.4.1
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Cookie Notices, <br/>AdGuard Popups, <br/>AdGuard Mobile App Banners, <br/>AdGuard Other Annoyances, <br/>AdGuard Widgets<br/><br/><b>Security:</b><br/>Online Malicious URL Blocklist<br/><br/><b>Language-specific:</b><br/>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Dutch, <br/>AdGuard Turkish, <br/>AdGuard Chinese, <br/>EasyList Italy
Other extensions used: | There no other extensions that are enabled. One extension thst is installed but disabled is ABP

### PR DESCRIPTION
**Pull request by zhelvis**
#176642

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -343,6 +343,7 @@ $xmlhttprequest,third-party,denyallow=linkvertise.com|fastly.net|statically.io|s
 !
 ! NOTE: Specific rules
 !
+gptchatly.com##div.container > aside
```
Comment (Sergey-Lyapin) [inline BaseFilter/sections/specific.txt:346]: I'd prefer
`gptchatly.com##div[id^="desktop-sidebar-"]`

Comment (zhelvis) [reply BaseFilter/sections/specific.txt:346]: There's also a mobile layout. Adds shows in the same aside elements
https://uploads.adguard.org/img_h7h6r.png

Comment (Sergey-Lyapin) [reply BaseFilter/sections/specific.txt:346]: Ok, your rule looks too wide, but it seems to make sense for this particular site.

**Review by Sergey-Lyapin — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
@zhelvis Please resolve conflicts.

**Pull request merged**